packages/airnode-protocol/contracts/rrp/AirnodeRrp.sol

```diff
@@ -83,7 +83,6 @@ contract AirnodeRrp is
         // If the Airnode address of the template is zero the template does not
         // exist because template creation does not allow zero Airnode address
         require(airnode != address(0), "Template does not exist");
-        require(fulfillAddress != address(this), "Fulfill address AirnodeRrp");
```

Comment:
We agreed that this is redundant in the current state in the post-audit communications